dice.py

```diff
@@ -14,7 +19,7 @@ def roll(self, count, sides):
 		if sides < 1:
 			return "You find it difficult to roll a die with {} sides.".format(sides)
 
-		if count > 10 or sides > 100000:
+		if count > self.config.get("max-count", 10) or sides > self.config.get("max-sides", 100000):
```

Comment:
might be good to have a constant for the defaults so we don't repeat arbitrary numbers